test/governance/GovernanceStaking.test.ts

```diff
@@ -342,13 +381,16 @@ describe("GovernanceStaking - staking with GD  and get Rewards in GDAO", () => {
     await governanceStaking
       .connect(staker)
       .transferFrom(founder.address, staker.address, "100");
-
+    const stakerProductivity = await governanceStaking.getProductivity(
+      staker.address
+    );
     expect(await governanceStaking.balanceOf(founder.address)).to.equal(0);
     expect(await governanceStaking.balanceOf(staker.address)).to.equal(100);
 
     expect((await governanceStaking.users(staker.address)).amount).to.equal(
       100
     );
+    expect(stakerProductivity[0]).to.be.equal("100");
```

Comment:
you didnt check stakerproductivity before the transfer

Reply:
Added this check as well